tests/e2e/cloudprovider/test-lb-service.sh

```diff
@@ -109,6 +151,37 @@ spec:
 EOF
 }
 
+########################################################################
+## Name: create_deployment
+## Desc: Makes sure the echoserver service Deployment is running
+## Params: None
+########################################################################
+function create_openstackcli_pod {
+    kubectl -n $NAMESPACE get pod | grep openstackcli | grep Running > /dev/null
+    if [[ $? -eq 1 ]]; then
```

Comment:
Is creating openstackcli pod required? Running openstack commands on VM where job is running is supported right?

Reply:
Currently, the lb-octavia CI job is running by talking to a k8s cluster running on catalyst cloud, for security reasons, the job has no openstack credential configured. The openstack credential is stored as a secret inside the cluster.

Reply:
Ah, got it . Didnt notice k8s cluster is on catalyst cloud.

Reply:
Because citynetwork doesn't support octavia.